lib/cli-compiler/Cargo.toml

```diff
@@ -0,0 +1,65 @@
+[package]
+name = "wasmer-compiler-cli"
+version = "2.2.1"
+description = "Wasmer Compiler CLI"
+categories = ["wasm", "command-line-interface"]
+keywords = ["wasm", "webassembly", "cli"]
+authors = ["Wasmer Engineering Team <engineering@wasmer.io>"]
+repository = "https://github.com/wasmerio/wasmer"
+license = "MIT"
+readme = "README.md"
+edition = "2018"
+default-run = "wasmer-compiler"
+build = "build.rs"
+
+[[bin]]
+name = "wasmer-compiler"
+path = "src/bin/wasmer_compiler.rs"
+doc = false
+
+[dependencies]
+wasmer-engine-universal-artifact = { version = "=2.2.1", path = "../universal-artifact", features = ["compiler"] }
+wasmer-compiler = { version = "=2.2.1", path = "../compiler" }
+wasmer-types = { version = "=2.2.1", path = "../types" }
+wasmer-vfs  = { version = "=2.2.1", path = "../vfs", default-features = false, features = ["host-fs"] }
```

Comment:
Why does wasmer-compiler-cli depend on the VFS? That's only used when running an instance, not compiling it.

Reply:
no specific reason, it's a leftover, I'll remove it.